src/GridView.php

```diff
@@ -451,10 +510,10 @@ private function renderTableBody(array $columns): string
             $rows[] = $this->renderTableRow($columns, $value, $key, $index);
 
             if ($this->afterRow !== null) {
-                /** @psalm-var array<array-key,string> */
+                /** @var string */
                 $row = call_user_func($this->afterRow, $value, $key, $index, $this);
 
-                if ($row !== []) {
+                if (!empty($row)) {
```

Comment:
Isn't using empty slower than explicitly comparing?

Reply:
`empty()` is a little faster than `===`. 

https://github.com/vjik/php-empty-vs-strict-equal-bench